### PR TITLE
#31 CSS applied to Commit View and Compare View

### PR DESCRIPTION
**Pull request by balawama**

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -20,7 +28,7 @@ function runShowMeTheCode(tabId, tab) {
     };
 
     if (isGitHub()) {
-        if (isBlobView() || pullRequestFileView()) {
+        if (isBlobView() || pullRequestFileView() || isCommitView() || isCompareView()) {
```
Comment (nowaklukasz) [inline src/background.js]: isPullRequestFileView

File: src/background.js [hunk 2]
```diff
@@ -20,7 +28,7 @@ function runShowMeTheCode(tabId, tab) {
     };
 
     if (isGitHub()) {
```
Comment (nowaklukasz) [inline src/background.js:30]: Is it worth having separate if for isGitHub ? you can as well squash it all into single if, with isGitHub as its first check.

File: src/background.js [hunk 3]
```diff
@@ -11,6 +11,14 @@ function runShowMeTheCode(tabId, tab) {
         return urlMatches(/.*\/pull\/.*\/files/);
     };
 
+    function isCommitView() {
+        return urlMatches(/.*\/commit\/.*/);
```
Comment (nowaklukasz) [inline src/background.js:15]: This is unlikely, but there is possibility that project name (i.e. Viridovix/commit/) will be caught by this regex

File: src/background.js (hunk 2 above)
Comment (nowaklukasz) [inline src/background.js:30]: Dismissed by https://github.com/Viridovix/show-me-the-code-github/pull/33#commitcomment-15086508

File: src/background.js (hunk 1 above)
Comment (balawama) [inline src/background.js]: ack

**Comment by nowaklukasz** [comment]
:+1:

File: src/background.js (hunk 3 above)
Comment (balawama) [inline src/background.js:15]: Alternative from that could be some check on DOM

Comment (nowaklukasz) [inline src/background.js:15]: /.*\/commit\/\w+$ ?

Comment (balawama) [inline src/background.js:15]: seems to be working

Comment (balawama) [inline src/background.js:15]: https://github.com/Viridovix/show-me-the-code-github/pull/34

**Pull request merged**